.github/DEVELOPMENT.md

```diff
@@ -0,0 +1,53 @@
+# Development Environment Setup
```

Comment:
this might need tweaking as it as a copy and paste from the `terraform-provider-aws` repo but wanted thoughts on adding it first before making the changes

Reply:
Can you remove this part for now? I'd like to just push the provider documentation changes and can follow up with improving the contribution docs.